test/e2e/e2e_test.go

```diff
@@ -74,6 +74,9 @@ var _ = ginkgo.BeforeSuite(func() {
 	_, err = ns.CreateNameSpace(c, constants.YurtE2ENamespaceName)
 	gomega.Expect(err).NotTo(gomega.HaveOccurred(), "fail to create namespace")
 
+	err = util.PrepareNodePoolWithNode(context.TODO(), yurtconfig.YurtE2eCfg.RuntimeClient, "openyurt-e2e-test-worker")
```

Comment:
For the sake of users or developers running e2e tests locally, I think the nodepool on the edge node needs to be destroyed during After Suite.

Reply:
The premise here is that now if the openyurt cluster wants to delete a nodepool with a node, it needs to expel the node from the cluster and rejoin it. Implementing this set of operations in e2e introduces a lot of additional complexity and may introduce other errors. So, as a alternative solution, I need to create a nodepool with nodes globally and let it be destroyed along with the cluster so that other e2e that require nodepool to have nodes can use it without conflict with other nodepool that does not need nodes.